projects/common/lib/components/replace-link-pipe/replacewithlinks.pipe.ts

```diff
@@ -0,0 +1,32 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+    name: 'replacewithlinks'
+})
+export class ReplacewithlinksPipe implements PipeTransform {
```

Comment:
Check out my comment on `replace-link-pipe/i18n/data/en/index.js`. Hardcoded links should be in applications, not in the library.  Sorry if I misunderstood you when we discussed this last.

Additionally, I think we can remove this pipe and suggest we put links in environments.ts files.  It worked really well for FPC.